tools/tool_settings.h

```diff
@@ -464,6 +466,55 @@ static std::string GetScreenshotDir(const gfxrecon::util::ArgumentParser& arg_pa
     return kDefaultScreenshotDir;
 }
 
+static void GetScreenshotSize(const gfxrecon::util::ArgumentParser& arg_parser, uint32_t& width, uint32_t& height)
+{
+    const auto& value = arg_parser.GetArgumentValue(kScreenshotSizeArgument);
+
+    if (!value.empty())
+    {
+        std::size_t x = value.find("x");
+        if (x != std::string::npos)
+        {
+            try
+            {
+                width  = std::stoul(value.substr(0, x));
+                height = std::stoul(value.substr(x + 1));
+            }
+            catch (std::exception& e)
```

Comment:
If any of these exceptions happened, that means we couldn't parse the screenshot size, right?  I think the user probably needs to be warned in each of these cases.